Justfile

```diff
@@ -132,7 +132,7 @@ version:
 # TODO: should unit test regeneration respect -f?
 @gen-data +mk_data_args="": (mk-data mk_data_args) gen-unit-data
 
-@mk-data +mk_data_args="": build-data-gen build-cli md
+@mk-data +mk_data_args="": build-data-gen build-cli (md mk_data_args)
```

Comment:
(unrelated line)
I didn't code review any of the Rust changes, but overall this looks like a nice improvement! It might be a little tricky since some of the paths have changed and if there's stuff like timestamps, but if we could diff the mocks/lockfiles generated with this change vs prior to this change, I would feel good about merging (once tests are passing)

Reply:
I diffed the lockfiles for the failing Python tests and found that they were identical when you compared the ones here vs. new ones generated on `main`. I didn't have much time to devote to that, but even some of the unit tests failed on `main` because of the Python version bump. I probably want to do a regen on `main`, fix those tests, then rebase this and continue to investigate.